index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/*" href="assets/images/portfolio.png">
+    <link rel="stylesheet" href="./assets/style/main.css">
+    <title>First Mobile version Portfolio</title>
+</head>
+<body>
+        <header class="header-content">
+            <span><a href="#"  id="mylogo" class="common-font-family">My Logo</a></span> 
+            <nav id="nav-ham-icon"><span id="hamburgur-icon">&#9776;</span></nav>
+        </header>
+        <main class="wrap-content">
+                <h1 id="myfullname" class="common-font-family">I’m Osman Wako
+                    Glad to see you!</h1>
+                <p class="common-font-family">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+                     If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
```

Comment:
- [ ] Same thing goes for your paragraph. Kindly  do the same.
- [ ] Kindly consider give your headline some line height to match the figma design